src/services/authService.js

```diff
@@ -29,4 +31,47 @@ export default class AuthService {
 
     return { token, ...Helper.omitFields(user, ['password']) };
   }
+
+  /**
+ *
+ * @param {string} email email address
+ * @returns {string} token
+ */
+  static async forgotPassword(email) {
+    const result = await User.findOne({ where: { email } });
+    if (!result) throw new Error('Email not found');
```

Comment:
Please always remember to leave a line after declaring a  variable or group of variables. There should be a line in between `41` and `42`, `42` and `43`,  `53` and `54` .
Let this style reflect in all your codebase Please...